### PR TITLE
AV-169470 : AKO Operator 1.9.1 Release and RH Certification

### PR DESCRIPTION
**Pull request by DixitAakash**
This PR includes changes for creating the AKO Operator release version 1.9.1, and certifying it through the RH portal.

### CODE REVIEW

File: ako-operator/api/v1alpha1/akoconfig_types.go [hunk 1]
```diff
@@ -183,7 +183,9 @@ type AKOConfigSpec struct {
 	// ImageRepository is where the AKO controller resides.
 	ImageRepository string `json:"imageRepository,omitempty"`
 	// ImagePullPolicy defines when the AKO controller image gets pulled.
-	ImagePullPolicy       string `json:"imagePullPolicy,omitempty"`
+	ImagePullPolicy string `json:"imagePullPolicy,omitempty"`
+	// ReplicaCount defines the number of replicas for AKO Statefulset
+	ReplicaCount          int `json:"replicaCount,omitempty"`
```
Comment (swathinsankaran) [inline ako-operator/api/v1alpha1/akoconfig_types.go:188]: @akshayhavile @DixitAakash Do we need to expose the `securityContext` and `podSecurityContext` also?

File: ako-operator/controllers/statefulset.go [hunk 2]
```diff
@@ -169,11 +169,12 @@ func BuildStatefulSet(ako akov1alpha1.AKOConfig, aviSecret corev1.Secret) (appsv
 	if err != nil {
 		return sf, err
 	}
-	var replicas int32 = 1
+	replicas := int32(ako.Spec.ReplicaCount)
```
Comment (swathinsankaran) [inline ako-operator/controllers/statefulset.go:172]: I think we have to limit the `replicaCount` to 2 always.

File: ako-operator/api/v1alpha1/akoconfig_types.go (hunk 1 above)
Comment (DixitAakash) [reply ako-operator/api/v1alpha1/akoconfig_types.go:188]: @swathinsankaran based on discussions we won't be exposing **securityContext** and **podSecurityContext** currently. @akshayhavile can you please confirm?

File: ako-operator/controllers/statefulset.go (hunk 2 above)
Comment (DixitAakash) [reply ako-operator/controllers/statefulset.go:172]: Added check to limit replica count to 2

**Review by swathinsankaran — APPROVED** [review]

**Review by akshayhavile — APPROVED** [review]

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Pull request merged**